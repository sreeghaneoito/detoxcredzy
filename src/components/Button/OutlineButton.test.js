import React from 'react';
import renderer from 'react-test-renderer';
import OutlineButton from './OutlineButton.component';

test('renders correctly', () => {
  const outlinebutton = renderer.create(<OutlineButton />).toJSON();
  expect(outlinebutton).toMatchSnapshot();
});
