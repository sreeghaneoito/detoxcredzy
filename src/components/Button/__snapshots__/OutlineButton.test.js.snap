// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "borderRadius": 4,
      "borderWidth": 1,
      "height": 48,
      "justifyContent": "center",
      "marginVertical": 10,
      "opacity": 1,
      "paddingHorizontal": 16,
    }
  }
  testID="undefinedbtn"
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 13,
          "fontWeight": "500",
          "textAlign": "center",
          "textTransform": "uppercase",
        },
        Object {
          "color": "#000000",
        },
      ]
    }
  />
</View>
`;
